Formula/k/kubeshark.rb

```diff
@@ -0,0 +1,20 @@
+class Kubeshark < Formula
+  desc "API Traffic Analyzer providing real-time visibility into Kubernetes network"
+  homepage "https://github.com/kubeshark/kubeshark"
+  url "https://github.com/kubeshark/kubeshark.git",
+    tag:      "v52.1.63",
+    revision: "68aabf262fdf6f3fe8f4aaef089f6f7c2471656e"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build-brew", "VER=v#{version}"
+    bin.install "kubeshark"
+  end
+
+  test do
+    output = shell_output("#{bin}/kubeshark version")
+    assert_equal "v#{version}", output.strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.